### PR TITLE
ref(ui): Change projects settings pagination to 50 items and remove grid emotion

### PR DESCRIPTION
**Pull request by davidenwang**
Only 567 orgs (<.1% of all sentry's orgs) have more than 50 projects which led to the decision to paginate this settings -> projects page (https://sentry.io/settings/sentry/projects/) at 50 items instead of 100. Also removed grid emotion. Thought I could be cool with some css grid stuff. But after trying both, I think flex makes more sense here, let me know if css grid would be preferred though.

**Before**
image

**After**
image

Closes SEN-1201

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx [hunk 1]
```diff
@@ -150,3 +154,19 @@ class OrganizationProjects extends AsyncView {
...
+`;
+
+const ContainerProjectListItem = styled('div')`
+  padding: ${space(2)};
+  flex: 1;
+`;
+
+const ContainerProjectStatsGraph = styled('div')`
+  padding: ${space(2)};
+  width: 25%;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx]: you might want a margin-left in the case of a really long project name

File: src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx [hunk 2]
```diff
@@ -150,3 +154,19 @@ class OrganizationProjects extends AsyncView {
...
+  align-items: center;
+  padding: 0;
+`;
+
+const ContainerProjectListItem = styled('div')`
+  padding: ${space(2)};
+  flex: 1;
+`;
+
+const ContainerProjectStatsGraph = styled('div')`
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx]: nit: We generally use `Container` as a suffix, also `Wrapper` might be better not to be confused with our "Container" components that do data fetching, etc.

File: src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx [hunk 3]
```diff
@@ -38,13 +39,15 @@ class OrganizationProjects extends AsyncView {
...
     const {orgId} = this.props.params;
+    const itemsPerPage = 50;
     return [
       [
         'projectList',
         `/organizations/${orgId}/projects/`,
         {
           query: {
             query: this.props.location.query.query,
+            per_page: itemsPerPage,
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx]: How are subsequent pages loaded?

File: src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx [hunk 4]
```diff
@@ -38,13 +39,15 @@ class OrganizationProjects extends AsyncView {
 
   getEndpoints() {
     const {orgId} = this.props.params;
+    const itemsPerPage = 50;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx]: nit: Can we move this to the top of the file and treat it as a constant?

File: src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx (hunk 3 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx]: Hmm yeah I think we need this? https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/asyncComponent.tsx#L245

Comment (davidenwang) [reply src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx]: upon further exploration with Billy it seems like the pagination component should take care of adding the query parameters for multiple pages. I also tested locally with ITEMS_PER_PAGE=2 and it functions as expected.

**Review by billyvg — APPROVED** [review]
:shipit:

File: src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx (hunk 3 above)
Comment (markstory) [reply src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx]: TIL about the auto created `${stateKey}PageLinks` properties.

**Review by markstory — APPROVED** [review]

**Pull request merged**